react-accordion-component/my-app/src/Accordion.js

```diff
@@ -0,0 +1,32 @@
+import './Accordion.css'
+import { useState } from 'react'
+/**
+ * Creates and accordion component
+ * Accepts an array of objects containing title and text
+ */
+export default function Accordion({ articles, }) {
+  const [current, setCurrent] = useState()
+  function handleCustomClick(i) {
+    if (i !== current) {
+    setCurrent(i)
+    } else {
+      setCurrent()
+    }
+  }
+
+  return (
+    articles.map((article, index) => {
+    return (<>
+      <div className="headers" key={index} onClick={() => handleCustomClick(index)}>
+        <h3>{article.title}</h3>
+        <button>{current === index ? 'Hide' : 'Show'}</button>
```

Comment:
This button doesn't do anything (it doesn't have an `onClick` and it's not in a form). Is it there just to show the action when you click the header?